.solhint.json

```diff
@@ -2,6 +2,9 @@
   "extends": ["solhint:recommended"],
   "plugins": ["prettier"],
   "rules": {
-    "indent": ["error", 2]
```

Comment:
deprecated in solhint@3.3.2 (https://github.com/protofire/solhint/blob/master/docs/rules.md)